<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyObjectDlg</class>
 <widget class="QDialog" name="CopyObjectDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Copy Object</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Copy Object Information</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Source</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QComboBox" name="mSlotsCombo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Slot list</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Slot status</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="3">
          <widget class="QLineEdit" name="mSessionText">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>240</green>
                 <blue>240</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Slot ID</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="mSlotIDText">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>240</green>
                 <blue>240</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Session</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QLineEdit" name="mLoginText">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>240</green>
                 <blue>240</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Source Object Information</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QComboBox" name="mSrcTypeCombo">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="mSrcLabelCombo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="mSrcObjectText">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>240</green>
                 <blue>240</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Target</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>Object Properties</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="0" column="0">
          <widget class="QLabel" name="label_26">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Label</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="mIDText"/>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="mUseRandCheck">
           <property name="text">
            <string>Use Random</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="mLabelText"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>Attributes</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="0" column="2">
          <widget class="QCheckBox" name="mSensitiveCheck">
           <property name="text">
            <string notr="true">Sensitive</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="mModifiableCheck">
           <property name="text">
            <string notr="true">Modifiable</string>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <widget class="QComboBox" name="mExtractableCombo"/>
         </item>
         <item row="2" column="5">
          <widget class="QComboBox" name="mVerifyCombo"/>
         </item>
         <item row="4" column="3">
          <widget class="QComboBox" name="mVerifyRecoverCombo"/>
         </item>
         <item row="1" column="5">
          <widget class="QComboBox" name="mDecryptCombo"/>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="mUnwrapCheck">
           <property name="text">
            <string notr="true">Unwrap</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="mUnwrapCombo"/>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="mSignRecoverCombo"/>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="mDeriveCheck">
           <property name="text">
            <string>Derive</string>
           </property>
          </widget>
         </item>
         <item row="4" column="5">
          <widget class="QDateEdit" name="mStartDateEdit"/>
         </item>
         <item row="3" column="4">
          <widget class="QCheckBox" name="mExtractableCheck">
           <property name="text">
            <string notr="true">Extractable</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QComboBox" name="mDeriveCombo"/>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="mSignRecoverCheck">
           <property name="text">
            <string>SignRecover</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="mSignCheck">
           <property name="text">
            <string notr="true">Sign</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QCheckBox" name="mVerifyCheck">
           <property name="text">
            <string notr="true">Verify</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QCheckBox" name="mWrapCheck">
           <property name="text">
            <string notr="true">Wrap</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QComboBox" name="mWrapCombo"/>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="mPrivateCombo"/>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="mTokenCombo"/>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="mTokenCheck">
           <property name="text">
            <string notr="true">Token</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QCheckBox" name="mDecryptCheck">
           <property name="text">
            <string notr="true">Decrypt</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QCheckBox" name="mTrustedCheck">
           <property name="text">
            <string>Trusted</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QCheckBox" name="mVerifyRecoverCheck">
           <property name="text">
            <string>VerifyRecover</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QComboBox" name="mTrustedCombo"/>
         </item>
         <item row="4" column="4">
          <widget class="QCheckBox" name="mStartDateCheck">
           <property name="text">
            <string>StartDate</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QComboBox" name="mEncryptCombo"/>
         </item>
         <item row="2" column="3">
          <widget class="QComboBox" name="mSignCombo"/>
         </item>
         <item row="5" column="1">
          <widget class="QDateEdit" name="mEndDateEdit"/>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="mEncryptCheck">
           <property name="text">
            <string notr="true">Encrypt</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="mSensitiveCombo"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="mModifiableCombo"/>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="mPrivateCheck">
           <property name="text">
            <string notr="true">Private</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="mEndDateCheck">
           <property name="text">
            <string>EndDate</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="mDestroyableCheck">
           <property name="text">
            <string>Destroyable</string>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QCheckBox" name="mCopyableCheck">
           <property name="text">
            <string>Copyable</string>
           </property>
          </widget>
         </item>
         <item row="5" column="5">
          <widget class="QComboBox" name="mCopyableCombo"/>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="mDestroyableCombo"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CopyObjectDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CopyObjectDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
